### PR TITLE
 Fix issue #4735 Text unit is set in dp instead of sp 

### PR DESCRIPTION
**Pull request by Vishwajith-Shettigar**
## Essential Checklist

- [ ] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [ ] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [ ] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [ ] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [ ] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [ ] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

## For UI-specific PRs only

If your PR includes UI-related changes, then:
- Add screenshots for portrait/landscape for both a tablet & phone of the before & after UI changes
- For the screenshots above, include both English and pseudo-localized (RTL) screenshots (see [RTL guide](https://github.com/oppia/oppia-android/wiki/RTL-Guidelines))
- Add a video showing the full UX flow with a screen reader enabled (see [accessibility guide](https://github.com/oppia/oppia-android/wiki/Accessibility-A11y-Guide))
- Add a screenshot demonstrating that you ran affected Espresso tests locally & that they're passing

### CODE REVIEW

File: app/src/main/res/values/themes.xml [hunk 1]
```diff
@@ -61,4 +61,9 @@
   <style name="OppiaActionBarTheme" parent="ThemeOverlay.MaterialComponents.Dark.ActionBar" >
     <item name="android:background">@color/component_color_shared_activity_action_bar_color</item>
   </style>
+
+  <style name="OppiaAdminpinTheme" parent="Theme.MaterialComponents.DayNight.NoActionBar" >
```
Comment (adhiamboperes) [inline app/src/main/res/values/themes.xml]: "OppiaAdminpinTheme" is not correctly formatted. "P"in should be in uppercase

File: gradle.properties
Comment (adhiamboperes) [inline gradle.properties:1]: You should remove this file from your commits.

File: app/src/main/res/layout/admin_pin_activity.xml [hunk 2]
```diff
@@ -13,10 +13,27 @@
...
+      android:layout_height="?attr/actionBarSize"
+      android:background="?attr/colorPrimary"
+      android:elevation="4dp"
+      style="@style/AdminpinToolbarTextAppearance"
+      app:titleTextAppearance="@style/AdminpinToolbarTextAppearance"
+      app:titleTextColor="@color/component_color_shared_activity_toolbar_text_color"
+      app:navigationIcon="@drawable/ic_close_white_24dp"
+      app:layout_constraintTop_toTopOf="parent" />
+
+    <!-- NestedScrollView below the Toolbar -->
```
Comment (adhiamboperes) [inline app/src/main/res/layout/admin_pin_activity.xml]: The comment is not really necessary here.

File: app/src/main/java/org/oppia/android/app/profile/AdminPinActivityPresenter.kt [hunk 3]
```diff
@@ -38,9 +38,10 @@ class AdminPinActivityPresenter @Inject constructor(
 
   /** Binds ViewModel and sets up text and button listeners. */
   fun handleOnCreate() {
-    activity.supportActionBar?.setDisplayHomeAsUpEnabled(true)
-    activity.supportActionBar?.setHomeAsUpIndicator(R.drawable.ic_close_white_24dp)
-    activity.supportActionBar?.setHomeActionContentDescription(R.string.admin_auth_close)
+
+//    activity.supportActionBar?.setDisplayHomeAsUpEnabled(true)
+//    activity.supportActionBar?.setHomeAsUpIndicator(R.drawable.ic_close_white_24dp)
+//    activity.supportActionBar?.setHomeActionContentDescription(R.string.admin_auth_close)
```
Comment (adhiamboperes) [inline app/src/main/java/org/oppia/android/app/profile/AdminPinActivityPresenter.kt]: It is very important for accessibility to set this ContentDescription value either here or in xml.

**Review by adhiamboperes — CHANGES_REQUESTED** [review]
Thanks @Vishwajith-Shettigar!

I think that this approach is generally great, and I would like for you to add the following screenshots to show that it works correctly:

- [ ]  Screenshots of the Admin Pin Screen
- [ ] Screenshots of the accessibility scanner showing that the issue is nolonger displayed.

File: app/src/main/res/values/themes.xml (hunk 1 above)
Comment (Vishwajith-Shettigar) [reply app/src/main/res/values/themes.xml]: <style name="OppiaAdminpinTheme" parent="Theme.MaterialComponents.DayNight.NoActionBar" > 
 
this seems redundant code so I deleted

**Comment by Vishwajith-Shettigar** [comment]
I will make a new pull request, sorry for the inconvenience

**Comment by adhiamboperes** [comment]
@Vishwajith-Shettigar please make sure your project builds locally everytime before pushing.

**Pull request closed without merging**